src/fs-plus.coffee

```diff
@@ -523,4 +542,18 @@ lstatSyncNoException ?= (args...) ->
 isPathValid = (pathToCheck) ->
   pathToCheck? and typeof pathToCheck is 'string' and pathToCheck.length > 0
 
+isMoveTargetValidSync = (source, target) ->
+  try
+    oldStat = fs.statSync(source)
```

Comment:
This line and the line below it should use `statSyncNoException` for speed :racehorse: 

Basically it removes the need for a `try`/`catch` and instead you can just return `true` if either `oldStat` or `newStat` is `false`.